lib/splash_screen.dart

```diff
@@ -43,9 +43,15 @@ class SplashScreen extends StatelessWidget {
                 alignment: AlignmentDirectional.topStart // or Alignment.topLeft
                 ),
             SizedBox(height: 30),
-            SpinKitFadingCube(
-              color: Colors.white,
-              size: 50.0,
+            Padding(
```

Comment:
Should only be 1 instead of 2 padding widgets here.

Reply:
Fixed